pkg/kopia/command/commands.go

```diff
@@ -1,486 +0,0 @@
-// Copyright 2022 The Kanister Authors.
```

Comment:
Since we are introducing this file in https://github.com/kanisterio/kanister/pull/1526 and removing it here, would it make more sense to merge these two PRs and then it would be easier to review only one version of these changes?